db/schema.rb

```diff
@@ -12,11 +12,6 @@
 
 ActiveRecord::Schema.define(version: 20170127000826) do
 
-  create_table "outcomes", force: :cascade do |t|
```

Comment:
Im still pretty sure you didnt mean to remove the `outcomes` table from your database schema here. You might want to kill this removal from this commit